packages/main/src/MultiComboBox.ts

```diff
@@ -456,9 +469,42 @@ class MultiComboBox extends UI5Element {
 	_itemToFocus?: IMultiComboBoxItem;
 	_itemsBeforeOpen: Array<MultiComboboxItemWithSelection>;
 	selectedItems: Array<IMultiComboBoxItem>;
-	FormSupport?: typeof FormSupportT;
 	static i18nBundle: I18nBundle;
 
+	get formValidityMessage() {
+		return MultiComboBox.i18nBundle.getText(FORM_MIXED_TEXTFIELD_REQUIRED);
+	}
+
+	get formValidity(): ValidityStateFlags {
+		const selectedItems = (this.items || []).filter(item => item.selected);
+
+		return { valueMissing: this.required && !this.value && !selectedItems.length };
+	}
+
+	async formElementAnchor() {
+		return this.getFocusDomRefAsync();
+	}
+
+	get formFormattedValue(): FormData | string | null {
+		const selectedItems = (this.items || []).filter(item => item.selected);
+
+		if (selectedItems.length) {
+			const formData = new FormData();
+
+			for (let i = 0; i < selectedItems.length; i++) {
+				formData.append(this.name, selectedItems[i].text);
+			}
+
+			if (this.value) {
+				formData.append(this.name, this.value);
```

Comment:
value should be first, easier to check for apps. also we should document that the first value is what the user typed and the rest (if any) is the selected